scripts/init.sh

```diff
@@ -52,8 +77,23 @@ if [[ -z "${SPARK_VERSION:-}" ]]; then
   export SPARK_BIN_VERSION=$(cd "$scripts/.."; mvn help:evaluate -Dexpression=spark.binary.version -q -DforceStdout)
 fi
 
+export CLASS_PATH="${CLASS_PATH:-local:///opt/spark/examples/jars/spark-examples_${SCALA_BIN_VERSION}-${SPARK_VERSION}.jar}"
+
 # check the Spark version is supported
 if ! [ -e "$root/src/main/scala-spark-$SPARK_BIN_VERSION" ]; then
     echo "This tool does not support Spark version ${SPARK_VERSION}."
     exit 1
 fi
+
+
+shift $((OPTIND - 1))
+FINAL_ARGS=()
+for arg in "$@"; do
+    FINAL_ARGS+=("$arg")
+done
+
+if [ ${#FINAL_ARGS[@]} -eq 0 ]; then
+    FINAL_ARGS+=("100")
+fi
+
+image_tag="$SPARK_VERSION-scala$SCALA_BIN_VERSION-java${JAVA_VERSION:-17}-ubuntu"
```

Comment:
I think we want to add the `-python3` to the `image_tag` if `true`:
```suggestion
if [ "$INCLUDE_PYTHON" == "true" ]; then WITH_PYTHON="-python3"; else WITH_PYTHON=""; fi
image_tag="$SPARK_VERSION-scala$SCALA_BIN_VERSION-java${JAVA_VERSION:-17}$WITH_PYTHON-ubuntu"
```
Then we can revert the Dockerfile changes again.